R/data_rana.R

```diff
@@ -0,0 +1,27 @@
+#'The Twins’ Magical Run Will Likely Go Through The Yankees. (Because Of Course.)
+#' 
+#' The raw data behind the story 
+#' "The Twins’ Magical Run Will Likely Go Through The Yankees. (Because Of Course.)"
+#' \url{https://fivethirtyeight.com/features/the-yankees-always-have-the-twins-number-will-this-postseason-be-different/}
+#' 
+#'   @format A data frame contains 98,796 rows representing Quasi-Win Shares (QWS), prorated to 162 team games, for every player from 1901 as of Sept. 27, 2019 and,  and 16 variables:
```

Comment:
Ensure the correct number of spaces before `@format` here; it's causing the help file when you run `?quasi_winshares` to come out wonky. It's a good idea to always read this help file after you finish this section to make sure everything looks good.